src/Aspire.Dashboard/Resources/Columns.resx

```diff
@@ -136,6 +136,9 @@
   <data name="SourceColumnDisplayContainerCommand" xml:space="preserve">
     <value>Command: {0}</value>
   </data>
+  <data name="SourceColumnSourceCopyFullPathToClipboard" xml:space="preserve">
+    <value>Copy File Path to Clipboard</value>
```

Comment:
I really don't like the casing here. It's just a sentence. Why is everything captalized? (I know you followed an existing pattern in other tooltips. I also think they're wrong).

What I'd like:
```suggestion
    <value>Copy file path to clipboard</value>
```

Also update the other copy tooltips to be cased like a regular sentence.

Reply:
For example, in GitHub tooltips have normal sentence capitalization:
![image](https://github.com/dotnet/aspire/assets/303201/c3740404-6a71-48ac-b50b-2f5101b92314)

Reply:
Yeah our style guide basically says sentence casing almost everywhere. We (I) did not do that consistently earlier.